frontend/src/static/js/v_summary.js

```diff
@@ -690,59 +704,74 @@ window.vSummary = {
         repo: user.repoName,
         name: user.displayName,
         location: this.getRepoLink(repo[index]),
+        repoIndex: index,
         totalCommits: user.totalCommits,
       });
     },
-    openTabZoomSubrange(user, repo, index) {
+    openTabZoomSubrange(user) {
       // skip if accidentally clicked on ramp chart
       if (drags.length === 2 && drags[1] - drags[0]) {
         const tdiff = new Date(this.filterUntilDate) - new Date(this.filterSinceDate);
         const idxs = drags.map((x) => x * tdiff / 100);
         const tsince = getDateStr(new Date(this.filterSinceDate).getTime() + idxs[0]);
         const tuntil = getDateStr(new Date(this.filterSinceDate).getTime() + idxs[1]);
-        this.openTabZoom(user, tsince, tuntil, repo, index);
+        this.openTabZoom(user, tsince, tuntil);
       }
     },
 
-    openTabZoom(user, since, until, repo, index) {
-      const { avgCommitSize } = this;
+    openTabZoom(user, since, until) {
+      const {
+        avgCommitSize, filterGroupSelection, filterTimeFrame, isMergeGroup, sortingOption,
+        sortingWithinOption, isSortingDsc, isSortingWithinDsc,
+      } = this;
 
       this.$emit('view-zoom', {
-        filterGroupSelection: this.filterGroupSelection,
-        avgCommitSize,
-        user,
-        location: this.getRepoLink(repo[index]),
-        sinceDate: since,
-        untilDate: until,
-        isMergeGroup: this.isMergeGroup,
+        zRepo: user.repoName,
```

Comment:
Can this function also be using a data structure? It's fine if not using one.

Reply:
You mean encompass all the properties into a zoomInfo object and only pass the zoomInfo object? Can ah

Reply:
Don't think its necessary to do this tho. There's only 3 parameters for this function.

Reply:
My bad, not looking clearly.